src/app/attendance/actions/attendanceActions.tsx

```diff
@@ -0,0 +1,26 @@
+"use server";
+import { AttendanceStatus, Table } from "@/types/index";
+import { getSupabaseClient } from "../../../../utils/supabase/supabaseClient";
```

Comment:
This is relatlive import. Let's use absolute paths. See the above one.

Reply:
FIx